app/src/main/res/layout/external_player_fragment.xml

```diff
@@ -37,6 +37,18 @@
             android:indeterminate="false"
             tools:progress="100"/>
 
+        <ImageButton
+            android:id="@+id/butPlay"
+            android:layout_width="52dp"
+            android:layout_height="52dp"
+            android:layout_alignParentRight="true"
+            android:layout_alignParentEnd="true"
+            android:layout_below="@id/episodeProgress"
+            android:layout_centerVertical="true"
+            android:contentDescription="@string/pause_label"
+            android:background="?attr/selectableItemBackground"
+            tools:src="@drawable/ic_play_arrow_white_36dp"/>
+
         <TextView
```

Comment:
This was giving me an error because of objects that use butPlay to determine their position being defined earlier